### DISCUSSION TITLE
Removing 'Unofficial modules not supported' precaution.

### DISCUSSION THREAD

**Discussion by Jdance-Media** [SmartlyDressedGames/Unturned-3.x-Community#4092] [Ideas]
Currently running the game with BattlEye prevents the loading of modules. Checking the Modules folder is an okay precaution, but for some users currently using my editor module it starts to become a hassle. It seems easier for them to just pick the correct launch option if they want to play multiplayer or not and not have to constantly install and reinstall the module.

**Comment by SDGNelson** [comment]
The BattlEye kick for unofficial modules? It's on the BE side, so I think there was some reason it was necessary.

**Reply by Jdance-Media** [reply]
Ah. Interesting. I remembered something about it being a precautionary measure of some kind (I think it was mentioned in another issue, not sure tho), but I wasn't sure if it was with the game or not. It had no localization when I checked so I probably should've assumed it was BE.

Do you think it would be something worth asking them to remove or should I just forget about it? It ain't really the biggest deal to move stuff out of the modules folder each time, just a minor inconvenience.

**Reply by SDGNelson** [reply]
I'm hesitant to poke them about it, sorry. For the meantime moving the modules to play on BE servers is the best workaround.

**Reply by DanielWillett** [reply]
Could we instead get a way to load a module with a launch option? (Like -Module "C:\Modules\Whatever.module") That way we could create shortcuts for stuff like that and not have to worry about battleye kicking when playing without it. Obviously this launch option would only work without battleye
